src/coreclr/jit/codegencommon.cpp

```diff
@@ -2097,6 +2097,26 @@ void CodeGen::genEmitUnwindDebugGCandEH()
     getDisAssembler().disAsmCode((BYTE*)*codePtr, finalHotCodeSize, (BYTE*)coldCodePtr, finalColdCodeSize);
 #endif // LATE_DISASM
 
+#ifdef DEBUG
+    if ((compiler->opts.disAsm || compiler->verbose) && compiler->opts.disAsmHexDumpFile)
+    {
+        FILE*   hexDmpf;
+        errno_t ec = _wfopen_s(&hexDmpf, compiler->opts.disAsmHexDumpFile, W("at")); // NOTE: file append mode
+        if (ec == 0)
+        {
+            assert(hexDmpf);
+
+            BYTE* addr = (BYTE*)*codePtr + compiler->GetEmitter()->writeableOffset;
+            for (unsigned int i = 0; i < codeSize; i++)
```

Comment:
is it possible to use [fwrite](https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/fwrite?view=msvc-170) passing it `addr` and `codeSize` without the for loop?

Reply:
I can look at it, I imagine it would be preferrable.

Reply:
There doesn't seem to be a way to write out the bytes in hex form for `fwrite`.